src/include/btmem.h

```diff
@@ -1126,6 +1125,15 @@ struct __wt_page_deleted {
     bool selected_for_write;
 };
 
+static inline wt_timestamp_t
+get_page_del_start_ts(WT_PAGE_DELETED *page_del)
+{
+    if (page_del->prepare_state == WT_PREPARE_INPROGRESS ||
```

Comment:
I switch from the macros to this function and `get_upd_start_ts` to trigger compiler errors on all the places that needed code change